cmd/commands/service.go

```diff
@@ -304,6 +331,33 @@ Additional curl arguments and flags may be specified after a double dash (--).`,
 	return command
 }
 
+// Print any HTTP errors in the given string to the given writer
+func PrintCurlHttpErrors(curlErrOutput string, w io.Writer) {
+	matched, err := regexp.MatchString("< HTTP/[\\d\\.]* 200", curlErrOutput)
+	if err != nil {
+		panic(err) // Should never happen unless the regex is invalid
+	}
+
+	if !matched {
+		for _, line := range strings.Split(curlErrOutput, "\n") {
+			matched, err := regexp.MatchString("< HTTP/[\\d\\.]* ", line)
+			if err != nil {
+				panic(err) // Should never happen unless the regex is invalid
```

Comment:
nit: what about using https://golang.org/pkg/log/#Panicf instead?

Reply:
No objections. Like I said, should never happen...